config/theme.yml

```diff
@@ -121,6 +121,51 @@ global_settings:
       width: 170
       height: 115
       scope: [stores]
+    default_80:
+      width: 80
+      height: 80
+      scope: [categories, products]
+    default_100:
+      width: 100
+      height: 100
+      scope: [manufacturers, suppliers]
+    default_120:
+      width: 120
+      height: 120
+      scope: [products]
+    default_160:
+      width: 160
+      height: 160
+      scope: [categories, products]
+    default_200:
+      width: 200
+      height: 200
+      scope: [stores, manufacturers, suppliers, products]
+    default_320:
+      width: 320
+      height: 320
+      scope: [categories, products]
+    default_400:
+      width: 400
+      height: 400
+      scope: [stores, products]
+    default_720:
+      width: 720
+      height: 720
+      scope: [products]
+    default_1000:
+      width: 1000
+      height: 1000
+      scope: [categories]
```

Comment:
```suggestion
      scope: [categories, products]
```

Reply:
Where would we need the 1000px on products?

Reply:
Product images modal